src/display/annotation_layer.js

```diff
@@ -2145,6 +2156,26 @@ class PopupElement {
     this.#container.style.zIndex =
       parseInt(this.#container.style.zIndex) - 1000;
   }
+
+  forceHide() {
+    this.#wasVisible = this.isVisible;
+    if (!this.#wasVisible) {
+      return;
+    }
+    this.#container.hidden = true;
+  }
+
+  restoreVisibility() {
```

Comment:
Naming things is hard, but should we perhaps call this `maybeShow` instead to more clearly indicate that it's the "inverse" of the `forceHide` method above?